SQL/0000-00-00-schema.sql

```diff
@@ -2186,3 +2186,81 @@ CREATE TABLE `media` (
   FOREIGN KEY (`session_id`) REFERENCES `session` (`ID`),
   FOREIGN KEY (`instrument`) REFERENCES `test_names` (`Test_name`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `issues`;
+CREATE TABLE `issues` (
+  `issueID` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `title` varchar(255) NOT NULL DEFAULT '',
+  `reporter` varchar(255) NOT NULL DEFAULT '',
+  `assignee` varchar(255) DEFAULT NULL,
+  `status` enum('new','acknowledged','feedback','assigned','resolved','closed') NOT NULL DEFAULT 'new',
+  `priority` enum('low','normal','high','urgent','immediate') NOT NULL DEFAULT 'low',
+  `module` int(10) unsigned DEFAULT NULL,
+  `category` enum('Behavioural Instruments','Behavioural Battery','Data Entry','Database Problems','Examiners','SubprojectID/Project/Plan Changes') DEFAULT NULL,
+  `dateCreated` datetime DEFAULT NULL,
+  `lastUpdate` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `lastUpdatedBy` varchar(255) DEFAULT NULL,
+  `sessionID` int(10) unsigned DEFAULT NULL,
+  `centerID` tinyint(2) unsigned DEFAULT NULL,
+  `candID` int(6) DEFAULT NULL,
+  PRIMARY KEY (`issueID`),
+  KEY `fk_issues_1` (`reporter`),
```

Comment:
Suggest using short format since it's way easier to read (maybe just my personal preference)
```
  PRIMARY KEY (`issueID`),
  FOREIGN KEY (`reporter`) REFERENCES `users` (`UserID`),
  FOREIGN KEY (`assignee`) REFERENCES `users` (`UserID`),
  FOREIGN KEY (`candID`) REFERENCES `candidate` (`CandID`),
  FOREIGN KEY (`sessionID`) REFERENCES `session` (`ID`),
  FOREIGN KEY (`CenterID`) REFERENCES `psc` (`CenterID`),
  FOREIGN KEY (`lastUpdatedBy`) REFERENCES `users` (`UserID`),
  FOREIGN KEY (`module`) REFERENCES `LorisMenu` (`ID`)
```